server/models/Document.js

```diff
@@ -242,9 +242,13 @@ Document.findByPk = async function (id, options = {}) {
 };
 
 type SearchResult = {
```

Comment:
This type is no longer named well, `SearchResponse` perhaps?

Reply:
yep, I'll change it.